libdrgn/symbol.h

```diff
@@ -34,6 +35,12 @@ struct drgn_symbol_result_builder {
 	};
 };
 
+#define _cleanup_symbol_ _cleanup_(drgn_symbol_cleanup)
+static inline void drgn_symbol_cleanup(void *p)
+{
+	drgn_symbol_destroy(*(struct drgn_symbol **)p);
```

Comment:
This can be type-safe:
```suggestion
static inline void drgn_symbol_cleanup(struct drgn_symbol **p)
{
	drgn_symbol_destroy(*p);
```
```